src/ng/directive/ngRef.js

```diff
@@ -0,0 +1,297 @@
+'use strict';
+
+/**
+ * @ngdoc directive
+ * @name ngRef
+ * @restrict A
+ *
+ * @description
+ * The `ngRef` attribute tells AngularJS to assign the controller of a component (or an element-directive)
```

Comment:
We can also assign attribute directives if we use `ngRefRead` so this sentence is not entirely accurate.